.github/workflows/build-bootc.yaml

```
@@ -50,7 +48,7 @@ jobs:
           file: bootc-agent-images/centos/Containerfile
           platforms: linux/amd64,linux/arm64
           push: true
-          tags: quay.io/flightctl/flightctl-agent-centos:bootstrap
+          tags: quay.io/${{ github.actor }}/flightctl-agent-centos:bootstrap
```

Just do this everywhere so that if it's updated at the top level we don't have to duplicate the changes. 
```suggestion
          tags: $QUAY_ORG/flightctl-agent-centos:bootstrap
```